linkis-engineconn-plugins/engineconn-plugins/spark/src/main/scala/org/apache/linkis/engineplugin/spark/executor/SparkPythonExecutor.scala

```diff
@@ -161,7 +161,7 @@ class SparkPythonExecutor(val sparkEngineSession: SparkEngineSession, val id: In
     val cmd = CommandLine.parse(pythonExec)
     cmd.addArgument(createFakeShell(pythonScriptPath).getAbsolutePath, false)
     cmd.addArgument(port.toString, false)
-    cmd.addArgument(EngineUtils.sparkSubmitVersion().replaceAll("\\.", ""), false)
```

Comment:
This bug can be modified by referring to the [ 0.11.0 method](https://github.com/apache/incubator-linkis/commit/0486a9abf3c2c2e1261195e41e758f5b9f0f9c78#diff-597ca2ca884fe266f156082c9c6a316435116a41c9ae56e8cbc9a56eea23fbee)